docs/pages/admin-guides/teleport-policy/integrations/azure-sync.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Discover Azure Access Patterns with Teleport Policy
+description: Describes how to import and visualize Azure subscription access patterns using Teleport Policy and Access Graph.
+---
+
+Teleport Policy streamlines and centralizes access management across your entire infrastructure. You can view access
+relationships in seconds, viewing unified, up-to-date relationships and policies between all users, groups, and
+computing resources.
+
+Teleport Policy with Access Graph offers insights into access patterns within your Azure subscription. By scanning
+users, groups, service principals, and role definitions, it provides a visual representation and aids in enhancing the permission model
+within your Azure environment. This functionality enables you to address queries such as:
+
+- What resources are accessible to Azure users and roles?
+- Which groups belong to other groups and how does that impact resource access?
+- How broad or narrow is scoping across the subscription when assigning roles to users and groups?
+
+Utilizing the Access Graph to analyze permissions within an Azure subscription necessitates the setup of the Access
+Graph (AG) service, a Discovery Service, and integration with your Azure subscription.
```

Comment:
```suggestion
To use Access Graph to analyze permissions within an Azure subscription, you need to set up the Access
Graph (AG) service, Teleport Discovery Service, and an integration with your Azure subscription.
```
Style suggestion

Reply:
Let's remove the (AG) as well. I don't think I've ever heard someone refer to it as "AG"